Source/WebGPU/WebGPU/RenderPassEncoder.mm

```diff
@@ -74,13 +75,14 @@
     if (queryIndex < m_visibilityResultBufferSize) {
         m_visibilityResultBufferOffset = queryIndex;
         [m_renderCommandEncoder setVisibilityResultMode:MTLVisibilityResultModeCounting offset:queryIndex];
+        m_queryBufferIndicesToClear.add(m_visibilityResultBufferOffset);
     }
 }
 
 static void setViewportMinMaxDepthIntoBuffer(auto& fragmentDynamicOffsets, float minDepth, float maxDepth)
 {
     static_assert(sizeof(fragmentDynamicOffsets[0]) == sizeof(minDepth), "expect dynamic offsets container to have matching size to depth values");
-    static_assert(startIndexForFragmentDynamicOffsets == 2, "code path assumes value is 2");
+    static_assert(startIndexForFragmentDynamicOffsets == 3, "code path assumes value is 3");
```

Comment:
This should probably be > 1, similar to how you wrote below, as the actual code didn't have to change